EncounterAPI/repository.bal

```diff
@@ -151,23 +174,28 @@ public isolated function search(map<string[]> searchParameters) returns r4:FHIRE
                 }
             }
             encounters = filteredList;
+            if encounters.length() > offset + count {
```

Comment:
I tested this, let's remove this IF condition

Reply:
done